siganushka/region-bundle/0.1/config/routes/siganushka_region.yaml

```diff
@@ -0,0 +1,2 @@
+siganushka_region:
+    resource: "@SiganushkaRegionBundle/Resources/config/routing/routes.xml"
```

Comment:
This path does not exist.

Reply:
My test has passed, can you provide details?